src/polyfills/details/details.js

```diff
@@ -30,6 +30,11 @@ var componentName = "wb-details",
 
 		if ( summary ) {
 			details = summary.parentNode;
+			Object.defineProperty( details, "open", {
```

Comment:
The purpose of this PR is to make the use of details summary uniform so that an application could use the polyfill and native implementation in the same way and this pattern is the only way to do this. @shawnthompson already needs this